src/workerd/jsg/dom-exception.c++

```diff
@@ -9,14 +9,16 @@
 namespace workerd::jsg {
 
 Ref<DOMException> DOMException::constructor(
-    Lock& js,
+    const v8::FunctionCallbackInfo<v8::Value>& args,
     Optional<kj::String> message,
     Optional<kj::String> name) {
-  kj::String errMessage = kj::mv(message).orDefault([&] { return kj::String(); });
-  return jsg::alloc<DOMException>(
-      kj::mv(errMessage),
-      kj::mv(name).orDefault([] { return kj::str("Error"); }),
-      js.v8Ref(v8::Exception::Error(v8Str(js.v8Isolate, errMessage)).As<v8::Object>()));
+  auto exception = jsg::alloc<DOMException>(
+      kj::mv(message).orDefault([&] { return kj::String(); }),
+      kj::mv(name).orDefault([] { return kj::str("Error"); }));
+  // Uses the built-in Error.captureStackTrace method to attach the stack
```

Comment:
Do other error types do this automatically? It's just our custom DOMException where we forgot to do this?

Reply:
Built-in errors do, yes. `DOMException` is the only JSG object we have that acts like an error. Before this change we are creating a separate error object and using its stack, which is just unnecessary if we use this built-in.